src/cli_args.rs

```diff
@@ -0,0 +1,36 @@
+use structopt::StructOpt;
```

Comment:
This allow us to parse the executable arguments, keeping the retro-compatibility with the `.env` and the environments vars.